src/views/create-account-view.ts

```diff
@@ -41,13 +42,76 @@ export class CreateAccountView extends LitElement
         return html`
             <ga-card class="card">
                 <ga-heading>Create an account</ga-heading>
-                <mwc-textfield class="input" outlined required label="User Name"></mwc-textfield>
-                <mwc-textfield class="mail" outlined required label="Mail"></mwc-textfield>
-                <mwc-textfield class="input" outlined required type="password" label="Password"></mwc-textfield>
-                <mwc-textfield class="input" outlined required type="password" label="Confirm password"></mwc-textfield>
-                <mwc-button id="createAccountButton" class="input" raised label="Submit"></mwc-button>
+                <mwc-textfield .value=${this.userName} @change=${this.updateUserName} class="input" outlined required label="User Name"></mwc-textfield>
+                <mwc-textfield .value=${this.userMail} @change=${this.updateUserMail} class="mail" outlined required label="Mail"></mwc-textfield>
+                <mwc-textfield .value=${this.userPass} @change=${this.updateUserPass} class="input" outlined required type="password" label="Password"></mwc-textfield>
+                <mwc-textfield .value=${this.userSecPass} @change=${this.updateUserSecPass} class="input" outlined required type="password" label="Confirm password"></mwc-textfield>
+                <mwc-button id="createAccountButton" @click=${this.createAccount} class="input" raised label="Submit"></mwc-button>
             </ga-card>
             <mwc-button id="loginButton" label="Already have an account?"></mwc-button>
         `;
     }
+
+    @state()
+    private userName = '';
+    @state()
+    private userMail = '';
+    @state()
+    private userPass = '';
+    @state()
+    private userSecPass = '';
+
+    private async createAccount(): Promise<void>
+    {
+        const user = {
+            name: this.userName,
+            password: this.userPass,
+            mail: this.userMail
+        };
+        try
+        {
+            const res = await fetch('http://localhost:5000/users', {
+                method: 'POST',
+                headers: {
+                    'Content-type': 'application/json'
+                },
+                body: JSON.stringify(user)
+            });
+
+            console.log( res ? `Account ${this.userName} created.` : 'Couldn\'t create an account.');
```

Comment:
No console outputs please. If error or success should be displayed to the user, do it properly by showing it in the UI. Or leave it away if it has no relevance for the user.